### PR TITLE
fix(footer): link/title correct font

### PR DESCRIPTION
**Pull request by mJarsater**
**Describe pull-request**  
This PR updates all the footer links to use the correct font (headline-07)

**Solving issue**  
Fixes: [CDEP-2839](https://tegel.atlassian.net/browse/CDEP-2839)

**How to test**  
1. Go to Footer
2. Add a top part (via controls)
3. Check the links and make sure they use headline-07

[CDEP-2839]: https://tegel.atlassian.net/browse/CDEP-2839?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by alexandereneroth — APPROVED** [review]
Looks good

**Review by theJohnnyMe — APPROVED** [review]
Nice catch!

**Comment by mJarsater** [comment]
Have you gotten the answers you need for design @TatyanaBumazhkina ? This one is waiting for your approval to be merged.

**Comment by TatyanaBumazhkina** [comment]
Tested and passed

**Pull request merged**
